service-registry/src/main/java/io/servicecomb/serviceregistry/client/http/ServiceRegistryClientImpl.java

```diff
@@ -547,6 +549,55 @@ public void watch(String selfMicroserviceId, AsyncResultCallback<MicroserviceIns
     return null;
   }
 
+  @Override
+  public FindInstancesResponse findServiceInstance(String consumerId, InstanceCache instanceCache) {
+    Holder<FindInstancesResponse> holder = new Holder<>();
+    IpPort ipPort = ipPortManager.getAvailableAddress(false);
+    
+    CountDownLatch countDownLatch = new CountDownLatch(1);
+    RestUtils.get(ipPort,
+        Const.REGISTRY_API.MICROSERVICE_INSTANCES,
+        new RequestParam().addQueryParam("appId", instanceCache.getAppId())
+            .addQueryParam("serviceName", instanceCache.getMicroserviceName())
+            .addQueryParam("version", instanceCache.getMicroserviceVersionRule())
+            .addQueryParam("rev", instanceCache.getRev())
+            .addHeader("X-ConsumerId", consumerId),
+        new Handler<RestResponse>() {
+          @Override
+          public void handle(RestResponse restResponse) {
+            HttpClientResponse response = restResponse.getResponse();
+            response.bodyHandler(
+                bodyBuffer -> {
+                  try {
+                    if (!(304 == response.statusCode())) {
+                      holder.value = JsonUtils.readValue(bodyBuffer.getBytes(), FindInstancesResponse.class);
+                    } else {
+                      holder.value = new FindInstancesResponse();
+                    }
+                  } catch (Exception e) {
+                    LOGGER.warn(bodyBuffer.toString());
+                    e.printStackTrace();
```

Comment:
liubao said print exception, not just print to console, it should be print to log like:
LOGGER.warn(xxx, e)